core/src/modules/DynamicsModule/include/BBMDynamics.hpp

```diff
@@ -12,6 +12,11 @@
 #include "include/MEBParameters.hpp"
 #include "include/IDynamics.hpp"
 
+#ifndef DGCOMP
+#define DGCOMP 3 // define to make red lines go away in the IDE
```

Comment:
Perhaps reword as "define to stop static analysis tools from raising unwanted errors"?

Reply:
I went with `// Define to prevent errors from static analysis tools`